File: springboot-validation/src/main/java/com/hoangtien2k3/service/UserService.java
package com.hoangtien2k3.service;

import com.hoangtien2k3.model.User;
import com.hoangtien2k3.repository.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;


@Service
public class UserService {

    @Autowired
    private UserRepository userRepository;

    public User createUser(User user) {
        return userRepository.save(user);
    }

}

File: Project-Spring-Boot/library-management-system/src/main/kotlin/com/hoangtien2k3/librarymanagementsystem/service/UserService.kt
package com.hoangtien2k3.librarymanagementsystem.service

import com.hoangtien2k3.librarymanagementsystem.entity.User
import com.hoangtien2k3.librarymanagementsystem.repository.UserRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class UserService {

    @Autowired
    private lateinit var userRepository: UserRepository

    fun findAll(): List<User> {
        return userRepository.findAll()
    }

    fun save(user: User): User {
        return userRepository.save(user)
    }
}